proof/crefine/ARM_HYP/Retype_C.thy

```diff
@@ -5574,7 +5574,7 @@ proof -
     apply (clarsimp simp: ko_vcpu_def vcpu0_def)
     apply (clarsimp simp: rf_sr_def cstate_relation_def carch_state_relation_def
                           cmachine_state_relation_def Let_def h_t_valid_clift_Some_iff)
-    apply (subgoal_tac "region_is_bytes p 456 \<sigma>'")
+    apply (subgoal_tac "region_is_bytes p 464 \<sigma>'")
```

Comment:
It's hard to tell from the diff, but can you use `?vcpusz` here as well?

Reply:
Yeah, we're still in reach of the lemma statement, does it not work?

Reply:
Let me check, I thought this was already a different lemma. Would definitely be nicer to use here if not.

Reply:
You are absolutely right. Will update.